modules/tactical/src/idb.ts

```diff
@@ -18,15 +19,23 @@ export interface Idb {
  * An in-memory implementation of the Idb interface, mostly useful for testing.
  */
 export class InMemoryIdb implements Idb {
-  db: Object = {};
+  db: Object;
+
+  /**
+   * Opens a connection to the Idb database.
+   */
+  open(database: string, stores: string[]) {
+    this.db = {};
+    stores.forEach((store: string) => { this.db[store] = {}; });
+  }
```

Comment:
Correct. To have two individual instances pointing to the same DB, you have to instantiate and open two separate objects. Obviously, not for this class specifically, but this class is a testing mock.